openpype/pipeline/colorspace.py

```diff
@@ -17,8 +20,56 @@
 log = Logger.get_logger(__name__)
 
 
-class CashedData:
+class CachedData:
     remapping = None
+    python3compatible = None
```

Comment:
Admittedly this isn't necessarily about being Py3 compatible - it's just about whether `PyOpenColorIO` can be imported or not in the current running python session

```suggestion
    has_compatible_ocio_package = None
```

Reply:
added via b8e054a50cac66c4e671da0261093be68db555b2